ci/scripts/jenkins_build.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+source ci/scripts/jenkins_common.sh
+
+gpuci_logger "Creating conda env"
+conda config --add pkgs_dirs /opt/conda/pkgs
+conda config --env --add channels conda-forge
+conda config --env --set channel_alias ${CONDA_CHANNEL_ALIAS:-"https://conda.anaconda.org"}
+mamba install -q -y -n base -c conda-forge "boa >=0.10"
+mamba create -q -y -n morpheus python=${PYTHON_VER}
+conda activate morpheus
+
+gpuci_logger "Installing CI dependencies"
+mamba env update -q -n morpheus -f ./docker/conda/environments/cuda${CUDA_VER}_ci.yml
+
+gpuci_logger "Check versions"
+python3 --version
+gcc --version
+g++ --version
+
+gpuci_logger "Check conda environment"
+conda info
+conda config --show-sources
+conda list --show-channel-urls
+
+gpuci_logger "Checking S3 cuDF cache"
+CONDA_BLD_DIR=/opt/conda/conda-bld
+CUDF_CONDA_COMMIT=$(git log -n 1 --pretty=format:%H -- ci/conda)
+CUDF_CONDA_CACHE_PATH="/cudf/${CUDA_VER}/${PYTHON_VER}/${RAPIDS_VER}/${CUDF_CONDA_COMMIT}/${NVARCH}/cudf_conda.tar.bz"
+CUDF_CONDA_CACHE_URL="${S3_URL}${CUDF_CONDA_CACHE_PATH}"
+CUDF_CONDA_TAR="${WORKSPACE_TMP}/cudf_conda.tar.bz"
+
+echo "Checking ${DISPLAY_URL}${CUDF_CONDA_CACHE_PATH}"
+set +e
+aws s3 cp --no-progress ${CUDF_CONDA_CACHE_URL} ${CUDF_CONDA_TAR}
+CUDF_CACHE_CHECK=$?
+set -e
+
+if [[ "${CUDF_CACHE_CHECK}" != "0" ]]; then
+      gpuci_logger "Cache miss, Building cuDF"
+      mkdir -p ${CONDA_BLD_DIR}
+      # The --no-build-id bit is needed for sccache
+      CONDA_ARGS="--no-build-id --output-folder ${CONDA_BLD_DIR} --skip-existing --no-test" ${MORPHEUS_ROOT}/ci/conda/recipes/run_conda_build.sh libcudf cudf
+
+      gpuci_logger "sccache usage for cudf build:"
+      sccache --show-stats
+      ZS=$(sccache --zero-stats)
+
+      gpuci_logger "Archiving cuDF build"
+      cd $(dirname ${CONDA_BLD_DIR})
+      tar cfj ${CUDF_CONDA_TAR} $(basename ${CONDA_BLD_DIR})
+      cd -
+      aws s3 cp --no-progress ${CUDF_CONDA_TAR} ${CUDF_CONDA_CACHE_URL}
+else
+      gpuci_logger "Cache hit, using cached cuDF"
+      cd $(dirname ${CONDA_BLD_DIR})
+      tar xf ${CUDF_CONDA_TAR}
+      cd -
+fi
+
+gpuci_logger "Installing cuDF"
+mamba install -q -y -c file://${CONDA_BLD_DIR} -c nvidia -c rapidsai -c conda-forge libcudf cudf
```

Comment:
`-c local` should work instead of the file://

```suggestion
mamba install -q -y -c local -c nvidia -c rapidsai -c conda-forge libcudf cudf
```